ext/node/polyfills/internal/primordials.mjs

```diff
@@ -12,6 +12,7 @@ export const ArrayPrototypeSlice = (that, ...args) => that.slice(...args);
 export const ArrayPrototypeSome = (that, ...args) => that.some(...args);
 export const ArrayPrototypeSort = (that, ...args) => that.sort(...args);
 export const ArrayPrototypeUnshift = (that, ...args) => that.unshift(...args);
+export const BigInt = globalThis.BigInt;
```

Comment:
Is `BigInt` not available in `primordials` from `ext:core/mod.js`? If so we should fix it 😬